IDEAS/Resources/Scripts/Dymola/Buildings/Components/Examples/RectangularZoneExternalSurfaces.mos

```diff
@@ -0,0 +1,4 @@
+simulateModel("IDEAS.Buildings.Components.Examples.RectangularZoneExternalSurfaces", startTime=0, stopTime=604800, method="Lsodar", tolerance=1e-06, resultFile="RectangularZoneExternalSurfaces");
```

Comment:
did you remove the old script?

Reply:
I renamed it. So normally it should work fine.

Reply:
ok!

Reply:
strange how unit tests pass despite that no reference results are included for these mos scripts..